src/hooks/api/newsletter.ts

```diff
@@ -1,18 +1,33 @@
 import getConfig from 'next/config';
-import { useMutation } from 'react-query';
 
-const addNewsletterSubscriber = async ({ email }: { email: string }) => {
+import { useAuthenticatedMutation } from '@/hooks/api/authenticated-query';
+import type { Headers } from '@/hooks/api/types/Headers';
+import { fetchFromApi } from '@/utils/fetchFromApi';
+
+const addNewsletter = ({
+  headers,
+  email,
+}: {
+  headers: Headers;
+  email: string;
+}) => {
   const { publicRuntimeConfig } = getConfig();
-  const response = await fetch(
-    `${publicRuntimeConfig.newsletterApiUrl}/${email}/${publicRuntimeConfig.newsletterEmailListId}`,
-    {
+
+  return fetchFromApi({
+    path: `${publicRuntimeConfig.newsletterApiUrl}/mailinglist/${publicRuntimeConfig.newsletterEmailListId}`,
```

Comment:
fetchFromApi is meant to fetch from the udb3 backend. It's not meant to pass a base url via path